packages/types-algosdk/index.d.ts

```diff
@@ -145,6 +181,20 @@ declare module 'algosdk' {
 
   export function logicSigFromByte (encoded: any): any;
 
+  export function makeApplicationClearStateTxn(from: any, suggestedParams: any, appIndex: any, appArgs: any, accounts: any, foreignApps: any, foreignAssets: any, note: any, lease: any, rekeyTo: any): any;
+
+	export function makeApplicationCloseOutTxn(from: any, suggestedParams: any, appIndex: any, appArgs: any, accounts: any, foreignApps: any, foreignAssets: any, note: any, lease: any, rekeyTo: any): any;
+
+	export function makeApplicationCreateTxn(from: any, suggestedParams: any, onComplete: any, approvalProgram: any, clearProgram: any, numLocalInts: any, numLocalByteSlices: any, numGlobalInts: any, numGlobalByteSlices: any, appArgs?: any, accounts?: any, foreignApps?: any, foreignAssets?: any, note?: any, lease?: any, rekeyTo?: any): any;
+
+	export function makeApplicationDeleteTxn(from: any, suggestedParams: any, appIndex: any, appArgs: any, accounts: any, foreignApps: any, foreignAssets: any, note: any, lease: any, rekeyTo: any): any;
+
+	export function makeApplicationNoOpTxn(from: any, suggestedParams: any, appIndex: any, appArgs: any, accounts: any, foreignApps: any, foreignAssets: any, note: any, lease: any, rekeyTo: any): any;
+
+	export function makeApplicationOptInTxn(from: any, suggestedParams: any, appIndex: any, appArgs?: any, accounts?: any, foreignApps?: any, foreignAssets?: any, note?: any, lease?: any, rekeyTo?: any): any;
+
+	export function makeApplicationUpdateTxn(from: any, suggestedParams: any, appIndex: any, approvalProgram: any, clearProgram: any, appArgs: any, accounts: any, foreignApps: any, foreignAssets: any, note: any, lease: any, rekeyTo: any): any;
```

Comment:
extra tabs (below as well)

Reply:
can you please go to `view file` and check ? I think you may have commented on previous commit or i don't know why it is showing previous commit file

Reply:
Ok

Reply:
`view file` is showing correct indentation.